thundra/thundra_agent.py

```diff
@@ -70,6 +80,15 @@ def wrapper(event, context):
             self.data['transactionId'] = transaction_id
 
             self.execute_hook('before:invocation', self.data)
+            signal.signal(signal.SIGALRM, self.timeout_handler)
+
+            if hasattr(context, 'get_remaining_time_in_millis'):
+                timeout_duration = context.get_remaining_time_in_millis() - self.timeout_margin
```

Comment:
Is this the same implementation we did in the golang? get_remaining_time_in_millis is giving the time remaining to given timeout to aws lambda. For example we can indicate lambda timeout as 500ms but this function might be called at 200ms of the execution and gives us the value of 300. In this case our warning might be fallacious to the user?

If the above example will never occur you can ignore it.

Reply:
looks like we do not have chance to get real timeout value from AWS. 

thanks.